couchbase/pom.xml

```diff
@@ -21,10 +21,32 @@
 		<module>util</module>
 	</modules>
 
+	<repositories>
+		<repository>
+			<id>spring-milestones</id>
+			<name>Spring Milestones</name>
+			<url>https://repo.spring.io/milestone</url>
+		</repository>
+		<repository>
+			<id>spring-libs-snapshot</id>
+			<name>Spring Snapshot Repository</name>
+			<url>https://repo.spring.io/libs-snapshot</url>
+		</repository>
+	</repositories>
+
+
+
 	<dependencies>
 		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-data-couchbase</artifactId>
+			<groupId>org.springframework.data</groupId>
+			<artifactId>spring-data-couchbase</artifactId>
+			<version>4.0.0.RC2</version>
+		</dependency>
+
+		<dependency>
+			<groupId>com.couchbase.client</groupId>
```

Comment:
this shouldn't be needed as 4.0.0 pulls it in transitively

Reply:
done